### PR TITLE
solves merge conflicts and updates sighting model to include picture related fields

### PR DESCRIPTION
**Pull request by girmaD**

### CODE REVIEW

File: controller/api-controller.js [hunk 1]
```diff
@@ -99,7 +102,7 @@ module.exports = function(app) {
     } else {
```
Comment (Myau5x) [inline controller/api-controller.js:102]: is this "else block" works when there is no "if" anywhere?

File: public/js/login.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ $(document).ready(function() {
           window.location.replace("/sightings");  ///Change this
           // If there's an error, log the error
         })
+
```
Comment (Myau5x) [inline public/js/login.js:38]: ? Why empty string? I'm ok with this , but it seems for me that catching error right after waas more easy to understand

**Review by Myau5x — COMMENTED** [review]
I have couple comments. 
And I not sure about handlebars changes, may be Nicole should look at this

File: public/js/login.js (hunk 2 above)
Comment (girmaD) [reply public/js/login.js:38]: i am deleting this request

**Comment by girmaD** [comment]
deleted the merge-request

**Pull request closed without merging**